modules/dedicated-cluster-install-deploy.adoc

```diff
@@ -60,28 +66,18 @@ If either operator does not appear as installed, to troubleshoot further:
 * Switch to the *Operators* → *Installed Operators* page and inspect
 the *Status* column for any errors or failures.
 * Switch to the *Workloads* → *Pods* page and check the logs in any Pods in the
-`openshift-logging` and `openshift-operators-redhat` projects that are reporting issues.
+`openshift-logging` project that is reporting issues.
 
 . Create a cluster logging instance:
```

Comment:
I might s/Create/Deploy

I might also clarify that the logging operator call the elastic search operators when you deploy the logging instance.

Reply:
How about Create and deploy? That seems like an accurate description of the steps that follow. As for the logging/elasticsearch, there is a note on line 16 that calls out this requirement, and step 1 and 2 are install both Operators.

Reply:
I think that if you clarify the note at line 16 that "Create and deploy" would be fine. I think it would be helpful to be explicit about one calling the other.